### PR TITLE
remove preview from tooltip for terminal

### LINKED ISSUE #166490: Terminal quick fix incorrectly says ctrl+enter to preview
image

Related: https://github.com/microsoft/vscode/issues/166489

### PR DESCRIPTION
**Pull request by meganrogge**
fix #166490

### CODE REVIEW

File: src/vs/platform/actionWidget/browser/actionList.ts [hunk 1]
```diff
@@ -129,7 +130,11 @@ class ActionItemRenderer<T extends IListMenuItem<IActionItem>> implements IListR
 		if (element.disabled) {
 			data.container.title = element.label;
 		} else if (actionTitle && previewTitle) {
-			data.container.title = localize({ key: 'label', comment: ['placeholders are keybindings, e.g "F2 to apply, Shift+F2 to preview"'] }, "{0} to apply, {1} to preview", actionTitle, previewTitle);
+			if (this._supportsPreview) {
+				data.container.title = localize({ key: 'label-preview', comment: ['placeholders are keybindings, e.g "F2 to apply, Shift+F2 to preview"'] }, "{0} to apply, {1} to preview", actionTitle, previewTitle);
+			} else {
+				data.container.title = localize({ key: 'label', comment: ['placeholders are keybindings, e.g "F2 to apply, Shift+F2 to preview"'] }, "{0} to apply", actionTitle);
```
Comment (Tyriar) [inline src/vs/platform/actionWidget/browser/actionList.ts:136]: comment could be improved now

File: src/vs/platform/actionWidget/browser/actionList.ts [hunk 2]
```diff
@@ -172,11 +178,7 @@ export class ActionList<T extends IActionItem> extends Disposable {
 			getHeight: element => element.kind === ActionListItemKind.Header ? this._headerLineHeight : this._actionLineHeight,
 			getTemplateId: element => element.kind
 		};
-
-		this._list = this._register(new List(user, this.domNode, virtualDelegate, [
-			new ActionItemRenderer<IListMenuItem<IActionItem>>(resolver, this._keybindingService),
-			new HeaderRenderer()
-		], {
+		this._list = new List(user, this.domNode, virtualDelegate, [new ActionItemRenderer<IListMenuItem<IActionItem>>(preview, resolver, this._keybindingService), new HeaderRenderer()], {
```
Comment (Tyriar) [inline src/vs/platform/actionWidget/browser/actionList.ts:181]: No more register?

File: src/vs/platform/actionWidget/browser/actionWidget.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ export const IActionWidgetService = createDecorator<IActionWidgetService>('actio
 export interface IActionWidgetService {
 	readonly _serviceBrand: undefined;
 
-	show(user: string, items: IListMenuItem<IActionItem>[], delegate: IRenderDelegate<any>, anchor: IAnchor, container: HTMLElement | undefined, actionBarActions?: readonly IAction[]): Promise<void>;
+	show(user: string, _supportsPreview: boolean, items: IListMenuItem<IActionItem>[], delegate: IRenderDelegate<any>, anchor: IAnchor, container: HTMLElement | undefined, actionBarActions?: readonly IAction[]): Promise<void>;
```
Comment (Tyriar) [inline src/vs/platform/actionWidget/browser/actionWidget.ts:37]: Can we remove the _ prefix in the interface?

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: src/vs/platform/actionWidget/browser/actionList.ts (hunk 2 above)
Comment (meganrogge) [reply src/vs/platform/actionWidget/browser/actionList.ts:181]: accident 👍🏼

**Review by Tyriar — APPROVED** [review]

**Pull request merged**